Src/VimCore/Modes_Command_CommandMode.fs

```diff
@@ -50,6 +50,7 @@ type internal CommandMode
                 command
 
         let lineCommand = _parser.ParseLineCommand command 
+        _vimData.LastLineCommand <- Some lineCommand

```

Comment:
Why set this here vs. line 60 where `LastCommandLine` is set. This would mean that within operations like `RunLineCommand` the values of `LastCommandLine` and `LastLineCommand` represented different commands.

Reply:
This will store `LineCommand.ParseError` in case of syntax errors. Is that intended? 

---
In reply to: [189437911](https://github.com/jaredpar/VsVim/pull/2141#discussion_r189437911) [](ancestors = 189437911)

Reply:
Gotcha on syntax error. 

---
In reply to: [189438568](https://github.com/jaredpar/VsVim/pull/2141#discussion_r189438568) [](ancestors = 189438568)